amazon-kinesis-client/src/main/java/software/amazon/kinesis/common/DiagnosticUtils.java

```diff
@@ -35,7 +35,7 @@ public class DiagnosticUtils {
     public static void takeDelayedDeliveryActionIfRequired(String resourceIdentifier, Instant enqueueTimestamp, Logger log) {
         final long durationBetweenEnqueueAndAckInMillis = Duration
                 .between(enqueueTimestamp, Instant.now()).toMillis();
-        if (durationBetweenEnqueueAndAckInMillis > MAX_TIME_BETWEEN_REQUEST_RESPONSE / 3) {
+        if (durationBetweenEnqueueAndAckInMillis > 11000) {
             // The above condition logs the warn msg if the delivery time exceeds 11 seconds.
```

Comment:
note that the code in `master` is actually incorrect relative to the comment, since this actually triggers for 20 seconds not 11.